base_view_inheritance_extension/__openerp__.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Extended view inheritance",
+    "version": "8.0.1.0.0",
+    "author": "Therp BV,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
```

Comment:
my choice is agpl. but for this specific module i see it would be good to have this available for the asocial part of the odoo community too. i would be damned to make such a commit myself, but if you submit a PR to make this LGPL i will accept it